DataPlotly/core/plot_types/radar.py

```diff
@@ -0,0 +1,108 @@
+"""
+Radar chart factory
+
+.. note:: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+"""
+
+import os
+from plotly import graph_objs
+from qgis.PyQt.QtGui import QIcon
+from DataPlotly.core.plot_types.plot_type import PlotType
+import plotly.colors as pc
+
+import numpy as np
+class RadarChartFactory(PlotType):
+    """
+    Factory for radar charts
+    """
+
+    @staticmethod
+    def type_name():
+        return 'radar'
+
+    @staticmethod
+    def name():
+        return PlotType.tr('Radar Plot')
+
+    @staticmethod
+    def icon():
+        return QIcon(os.path.join(os.path.dirname(__file__), 'icons/radar.svg'))
+
+    @staticmethod
+    def create_trace(settings):
+
+        if len(settings.y_radar_values) == 0:
+            return []
+
+        x = settings.properties["y_fields_combo"].split(", ")
+
+        # Sample colors from the color scale based on the length of settings.y
+        colors_list = pc.sample_colorscale(settings.properties['color_scale'], np.linspace(0, 1, len(settings.y_radar_values[0])))
+        # List repeating the line type for each element in settings.y
+        line_type_list = [settings.properties['line_dash']] * len(settings.y_radar_values)
```

Comment:
This is the same line_type for every trace, we don't need to have a list.

We can use the line type in the `for` loop.

Reply:
In fact we need it, because if threshold is used, we add another line type at the end of the line_type_list